tribe-ext-event-progress-bar.php

```diff
@@ -0,0 +1,268 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Event Progress Bar
+ * Plugin URI:        https://theeventscalendar.com/extensions/event-progress-bar/
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-event-progress-bar
+ * Description:       This experimental extension adds a progress bar below the event description in list view and day view (V2).
+ * Version:           1.0.0
+ * Extension Class:   Tribe\Extensions\EventProgressBar\Main
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       tribe-ext-event-progress-bar
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+namespace Tribe\Extensions\EventProgressBar;
+
+use Tribe__Extension;
+
+/**
+ * Define Constants
+ */
+
+if ( ! defined( __NAMESPACE__ . '\NS' ) ) {
+	define( __NAMESPACE__ . '\NS', __NAMESPACE__ . '\\' );
+}
+
+// Do not load unless Tribe Common is fully loaded and our class does not yet exist.
+if (
+	class_exists( 'Tribe__Extension' )
+	&& ! class_exists( NS . 'Main' )
+) {
+	/**
+	 * Extension main class, class begins loading on init() function.
+	 */
+	class Main extends Tribe__Extension {
+
+		/**
+		 * Is Events Calendar PRO active. If yes, we will add some extra functionality.
+		 *
+		 * @return bool
+		 */
+		public $ecp_active = false;
+
+		/**
+		 * Is Event Tickets active. If yes, we will add some extra functionality.
+		 *
+		 * @return bool
+		 */
+		public $et_active = false;
+
+		/**
+		 * Setup the Extension's properties.
+		 *
+		 * This always executes even if the required plugins are not present.
+		 */
+		public function construct() {
+			$this->add_required_plugin( 'Tribe__Events__Main', '5.0' );
```

Comment:
Add comment why the version you required.

Reply:
Done.